src/Homework06/Main2.java

```diff
@@ -0,0 +1,28 @@
+package Homework06;
+
+public class Main2 {
+
+    /**
+     * процедура принимающая на вход массив из целых чисел, и выводящая в консоль значимые числа влево, а нули вправо.
+     * @param array - входящий массив целых чисел.
+     */
+    public  static void autoSorting(int[] array){
+        int[] outputArray = new int[array.length];
+        int indexOutputArray = 0;
+        for (int i = indexOutputArray; i < array.length; i++) {
```

Comment:
Очень не очень. Тебе надо пройти от начала до конца массив переданный в аргументы функции, поэтому не выдумывай переприсваивание и просто иди от ноля до конца массива. Присваивание из индекса другого массива вносит путаницу.
```suggestion
        for (int i = 0; i < array.length; i++) {
```

Reply:
Понял о чем ты.  Исправил.